pkg/apis/ceph.rook.io/v1/placement.go

```diff
@@ -90,6 +92,17 @@ func (p Placement) mergeNodeAffinity(nodeAffinity *v1.NodeAffinity) *v1.NodeAffi
 	return result
 }
 
+func (p Placement) mergeTolerations(toleration []v1.Toleration) []v1.Toleration {
+	// no toleration is specified yet, return placement's toleration
+	result := p.Tolerations
+	if toleration == nil {
+		return result
```

Comment:
No need to assign to a temp variable `result` if you just return this, right?
```suggestion
		return p.Tolerations
```

Reply:
done